torchrl/objectives/ppo.py

```diff
@@ -454,6 +511,9 @@ def forward(self, tensordict: TensorDictBase) -> TensorDictBase:
         if self.critic_coef:
             loss_critic = self.loss_critic(tensordict)
             td_out.set("loss_critic", loss_critic.mean())
+        if self.kl_coef:
+            loss_kl_init = self.kl_coef * self._kl_init(dist, tensordict)
+            td_out.set("loss_kl_init", loss_kl_init.mean())
         td_out.set("ESS", ess.mean() / batch)
         return td_out
 
```

Comment:
detail what is new/old policy in the following docstring

Reply:
KLPENPPOLoss one. (you haven't edit that so couldn't comment on those lines)

Reply:
also is KL(old_policy, new_policy) correct? here they use the opposite:  https://github.com/lvwerra/trl/blob/main/trl/trainer/ppo_trainer.py#LL960C28-L960C39

Reply:
![image](https://github.com/pytorch/rl/assets/25529882/687d7501-a269-4dcb-b3d0-254e045adc45)
From the paper